src/DocumentFormat.OpenXml.Framework/Validation/DocumentValidator.cs

```diff
@@ -106,6 +107,20 @@ private void ValidatePart(OpenXmlPart part, ValidationContext context)
                     {
                         Validate(context);
                     }
+                    else if (part.Uri.ToString().EndsWith(".xml", System.StringComparison.InvariantCultureIgnoreCase) &&
```

Comment:
need to dispose the stream from getpart. probably best to add an extension method of ".IsEmptyPart()" that can handle correctly disposing the stream

Reply:
Done